src/renderer/constants/dutyStatus.ts

```diff
@@ -0,0 +1,12 @@
+export enum DutyStatus {
+    // dont change order of values
+    unknown,
+    scheduled,
+    missed,
+    orphaned,
+    proposed,
+    attested,
+}
+
+const text = ["Unknown", "Scheduled", "Missed", "Orphaned", "Proposed", "Attested"];
```

Comment:
It's a lot safer to have map DutyStatus => text but that said, enum can also have those text values

Reply:
then I will use map